CONTRIBUTING.md

```diff
@@ -1,5 +1,8 @@
 # Contributing
 
+This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to uphold this code.
```

Comment:
Hey @mojavelinux, thanks for the feedback. That language actually comes from http://todogroup.org/opencodeofconduct ([source](https://github.com/todogroup/opencodeofconduct/blob/gh-pages/_layouts/code-of-conduct.html#L42-L45)). Would you mind opening an issue/pull request to discuss your suggestion over there?

What I like about "uphold" is that it is active instead of passive. I'm all for tweaking it though, so looking forward to discussing it.